lib/widgets/breez_avatar_dialog.dart

```diff
@@ -11,147 +12,180 @@ import 'package:image/image.dart' as DartImage;
 import 'package:image_cropper/image_cropper.dart';
 import 'package:image_picker/image_picker.dart';
 
+import 'flushbar.dart';
+
 int scaledWidth = 200;
 var _transparentImage = DartImage.Image(scaledWidth, scaledWidth);
 
 Widget breezAvatarDialog(BuildContext context, UserProfileBloc userBloc) {
   AutoSizeGroup _autoSizeGroup = AutoSizeGroup();
   BreezUserModel _currentSettings;
+  List<int> _imageData;
 
   final _nameInputController = TextEditingController();
   userBloc.userPreviewStream.listen((user) {
     _nameInputController.text = user.name;
     _currentSettings = user;
   });
 
-  Future _pickImage(BuildContext context) async {
+  Future _pickImage(BuildContext context, setState) async {
```

Comment:
Instead of passing `setState` why not return the file url as the future result? Mutating the state can be done by the caller.